pkgs/flox/default.nix

```diff
@@ -126,6 +128,11 @@ in
       pname = cargoToml.package.name;
       version = envs.FLOX_VERSION;
       src = flox-src;
+      postPatch = ''
+        substituteInPlace crates/flox/src/utils/init/gitConfig.in \
+	  --replace __flox_gh__ ${flox-gh}/bin/flox-gh \
+	  --replace __gh__ ${gh}/bin/gh
+      '';
```

Comment:
```suggestion
```

Reply:
add `GH_BIN` and `FLOX_GH_BIN` in under [line 42](https://github.com/flox/flox/pull/184/files#diff-c6844984ed543dda6cd59e21f0951372a0939287983b5d45bfe5827593c97393R41) instead

Reply:
Applied in separate commit. :+1: